frontend/lib/resolution/scope-queries.cpp

```diff
@@ -769,44 +765,34 @@ convertLimitations(Context* context, const VisibilityClause* clause) {
 // anchor errors.
 // 'isFirstPart' is true for A in A.B.C but not for B or C.
 // Returns nullptr in the event of an error.
-static const Scope* findScopeViz(Context* context,
-                                 const Scope* scope,
+static const Scope* findScopeViz(Context* context, const Scope* scope,
                                  UniqueString nameInScope,
                                  const ResolvedVisibilityScope* resolving,
-                                 ID idForErrs,
-                                 VisibilityStmtKind useOrImport,
+                                 ID idForErrs, VisibilityStmtKind useOrImport,
                                  bool isFirstPart) {
-
   // lookup 'field' in that scope
   std::vector<BorrowedIdsWithName> vec;
-  bool got = lookupInScopeViz(context, scope, resolving,
-                              nameInScope, useOrImport, isFirstPart, vec);
+  bool got = lookupInScopeViz(context, scope, resolving, nameInScope,
+                              useOrImport, isFirstPart, vec);
 
   if (got == false || vec.size() == 0) {
-    if (useOrImport == VIS_USE)
-      context->error(idForErrs, "could not find target of 'use'");
-    else
-      context->error(idForErrs, "could not find target of 'import'");
-
-    return nullptr;
-
-  } else if (vec.size() > 1 || vec[0].numIds() > 1) {
-    if (useOrImport == VIS_USE)
-      context->error(idForErrs, "ambiguity in finding target of 'use'");
-    else
-      context->error(idForErrs, "ambiguity in finding target of 'import'");
-
+    CHPL_REPORT(context, UnknownUseImport, idForErrs, useOrImport,
```

Comment:
You're right; this is an error that I intentionally removed because I wrongly thought it was unreachable based on testing with `chpl --dyno` instead of `testInteractive`. When I went back and corrected that mistake I missed this one, so I'll go figure out how to reach it now.

Reply:
Upon further review (and trying examples with `testInteractive`) I still believe it is unreachable, as if two modules are defined with the same name in the same scope that results in an error prior to here. I've already converted the ambiguity if-condition to an assertion, and I don't think there is anything further to do here; let me know if you disagree.